ui/nuxeo-user-group-management/nuxeo-create-user.js

```diff
@@ -214,18 +214,18 @@ import '../nuxeo-button-styles.js';
         })
         .catch(async (error) => {
           let message = error;
-          if (error.message) {
-            ({ message } = error);
-          } else if (error.response) {
+          if (error.response) {
             const text = await error.response.text();
             if (text) {
               try {
-                const json = JSON.parse(text);
-                this.errors = json.message;
+                const errorJSON = JSON.parse(text);
+                ({ message } = errorJSON);
```

Comment:
Remove both lines and specify it in single line as below.
                ({ message } = JSON.parse(text));